### DISCUSSION TITLE
how to show listen count?

### DISCUSSION THREAD

**Discussion by hunderslash** [audacious-media-player/audacious#68] [General]
I see the last.fm option and scrobbling works fine, I just don't see an option to show the listen count as a column. Is this possible?

**Comment by jlindgren90** [comment]
This isn't currently possible. Audacious doesn't remember how often you listen to a track.

**Reply by Maeda1** [reply]
Hello,
I have write (for my need) [those scripts](https://codeberg.org/Maeda/Audacious-plugin-track-count) (in Bash, for Linux) based on the Audacious's _track change_ plugin. Maybe that can help somehow.
